src/components/EditableTypography/EditableTypography.tsx

```diff
@@ -155,7 +165,10 @@ const EditableTypography: VibeComponent<EditableTypographyProps, HTMLElement> =
           <AnimatedTypography
             ref={typographyRef}
             aria-hidden={isEditing}
-            className={cx(styles.typography, typographyClassName, { [styles.hidden]: isEditing })}
+            className={cx(styles.typography, typographyClassName, {
+              [styles.hidden]: isEditing,
+              [styles.disabled]: readOnly
```

Comment:
was that a missing class? not related to the current feature?

Reply:
Yes, the class was there but the condition was missing